test-site/src/pages/PeoplePage.tsx

```diff
@@ -28,6 +28,7 @@ export function PeoplePage() {
 
   return (
     <div>
+      <SearchBar />
```

Comment:
nit: I think this was just for testing purposes?

Reply:
ah! Thanks.